continuous_integration/environment-3.9.yaml

```diff
@@ -23,9 +23,7 @@ dependencies:
   - pytables
   - zarr
   # `tiledb-py=0.17.5` lead to strange seg faults in CI.
```

Comment:
Looks like this comment is not longer needed? Or at least looks strange with the current version pin 

```suggestion
```